src/modules/modal.js

```diff
@@ -0,0 +1,91 @@
+import getshow from './getShows.js';
+import { getCommentsData } from './getComments.js';
+import { commentTemplate } from './addComment.js';
+
+const modal = document.getElementById('modal-popup');
+const modalTemplate = async (show) => {
+  modal.innerHTML = '';
+  modal.id = show.id;
+
+  const comments = await getCommentsData(show.id);
+
+  const closeSign = document.createElement('i');
+  closeSign.id = 'close';
+  closeSign.className = 'fas fa-times hvr-pulse';
+  modal.appendChild(closeSign);
+  modal.appendChild(closeSign);
+
+  const showInfo = document.createElement('div');
+  showInfo.className = 'showList';
+
+  const showImg = document.createElement('img');
+  showImg.className = 'modal-img';
+  showImg.alt = 'Show Image';
+  showImg.src = show.image.medium;
+  showInfo.appendChild(showImg);
+
+  const showDescrpt = document.createElement('div');
+  showDescrpt.className = 'showContent';
+
+  const showName = document.createElement('p');
+  showName.className = 'show-title';
+  showName.textContent = show.name;
+  showDescrpt.appendChild(showName);
+
+  const showSummary = document.createElement('p');
+  showSummary.className = 'show-summary';
+  showSummary.textContent = show.summary;
+  showDescrpt.appendChild(showSummary);
+  showInfo.appendChild(showDescrpt);
+  modal.appendChild(showInfo);
```

Comment:
- [ ] (Optional) It would be nice to remove the `HTML` tags that show on each popup. This would improve the look and the popup section. 😄